_pages/openhours_about.md

```diff
@@ -50,24 +50,19 @@ We look forward to talking with you.
 
 # The Team 
 
-<div class="container">
-<div class="row">
 <div class="col-sm-3">
-<div class="openhours-about-image" style="background-image:url({% asset_path 'robert-small.jpg' %});"></div>
+<div class="openhours-about-image" style="background-image:url({% asset_path 'sahaj-small.jpg' %});"></div>
 <ul class="name-and-title text-center">
-<li>Robert Wolf</li>
+<li>Sahaj Sarup</li>
 <li>OpenHours Host</li>
 </ul>
 </div>
 <div class="col-sm-9" markdown="1">
-Robert Wolff is a technical writer, open source evangelist, community leader and engineer with a history of working in and around esteemed academic institutions and STEAM (science, technology, engineering, art and mathematics) based educational programs. Right now, Robert works as community manager for 96Boards at Linaro. He maintains several of 96Boards’ Open Source orgs and repositories, runs a variety of community driven projects/initiatives, and hosts his own live, weekly online video “podcast” dubbed “96Boards OpenHours”.
-
-Robert graduated from University of California, San Diego with a bachelor's degree in Electrical Engineering. During his undergrad, he was heavily involved with the Global Teams in Engineering Services (TIES) program. He participated in several humanitarian engineering projects, working in multi-disciplinary teams, to deliver useful prototypes to promote STEM education for underprivileged communities around his hometown of San Diego, California. It was during this time Robert began to develop an interest in open source, embedded systems, and IoT. 
-
-Prior to his work with Linaro/96Boards, he worked for the Qualcomm Institute at UCSD creating an extensive online specialization for the DragonBoard 410c. These six courses are hosted by Coursera.org and contain upward of 500 educational videos.
+Sahaj has a background in Computer Applications engineering and have previously worked as an intern at SUSE. Before Linaro, he was a major contributor to the android-rpi, project that aimed to provide AOSP support for the Raspberry Pi, and also was the owner and maintainer of the official forums for the same. In his free time, he likes to Vlog (Video Blog) about his adventures in the world of Open Source and SBCs on his YouTube Channel (Geek Till it Hertz), where he likes to discuss new ARM hardware and unusual operating systems.
 </div>
```

Comment:
```suggestion

```
Duplicated? There is another one towards the bottom of the page rendering [http://staging-96boards-org-1164.websitepreview.linaro.org/openhours/about/](http://staging-96boards-org-1164.websitepreview.linaro.org/openhours/about/)